### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by erp78** [Austin-Li-1123/Tesla_forcasting_feature_analysis#13]
This project is about predicting Tesla stock prices using external factors such as competitor prices and the sentiments of Elon Musk's tweets. Three datasets are used - competitors stock prices, Tesla stock prices, and Twitter data. They consider the regression problem of predicting the next day stock price, and the classification problem of predicting whether the stock went up or down.

One thing I like about this project is how you conducted sentiment analysis on Elon Musk's tweets. I think this was a cool feature to include on top of the normal open and close prices, and it was interesting to consider what effect it would have on the Tesla stock price. Next, I think the section on avoiding overfitting was done well, and you describe in a clear way why you selected the polynomial degree that you did, especially with the graph. Additionally, I like how you plotted the true vs predicted graphs for each of your regression models, as it allowed us to visualize and compare their performances.

For areas of improvement, I think that some of your statements are too broad and you could've gone into deeper analysis. For example, instead of just saying that the plot shows a trend between tones and stock price, it would have been more informative to detail what that relationship is. Next, I would've liked to hear about your decision making process in selecting what models to fit - for example why did you choose a Random Forest? Did you do any hyperparameter tuning? Lastly, it would've been good if you laid out more clearly which features and labels were being used for each model, since you mention that you created some of your own and also changed which combinations of features were being used which makes it a little confusing.
